src/ZoneServer/Skills/Handlers/Scout/ObliqueFire.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using Melia.Shared.L10N;
+using Melia.Shared.Tos.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using Yggdrasil.Extensions;
+using static Melia.Zone.Skills.SkillUseFunctions;
+
+namespace Melia.Zone.Skills.Handlers.Scout
+{
+	/// <summary>
+	/// Handles the Scout skill Oblique Fire.
+	/// </summary>
+	[SkillHandler(SkillId.Scout_ObliqueFire)]
+	public class ObliqueFire : ITargetSkillHandler
```

Comment:
This handler doesn't seem to include the buff/debuff. Our current implementation of it is almost certainly incorrect, but at the very least the buff should be started. Ideally it would also be fixed up to do what it's supposed to do.